Training web/Task08-ReactJs/my-app/src/styles/_settings/_colors.scss

```diff
@@ -0,0 +1,15 @@
+$border-color : #D8D8D8;
+
+$button-background : black;
+$button-text: white;
+
+$footer-background: black;
+$footer-text: white;
+
+$nav-background:black;
+$nav-icon:white;
+$nav-text:white;
+
+
+$search-bar: #8E8E8E;
+$search-background: white;
```

Comment:
Aquí hay varias cosas.. yo personalmente haría una lista de colores pero con los nombres de los colores. 

En este caso blanco, negro y el gris (#D8D8D8). Así si los diseñadores cambian el tono del blanco solo se cambia en una sola línea. 

Con los nombres de los colores, generalmente uso [este tool](https://chir.ag/projects/name-that-color/#D8D8D8), donde ponés el código del color y te tira el nombre. Porque no me gusta que a veces pasa que empiezan a hacer esto:

- lightGray
- darkGray
- gray
- darkerGray

---------

Ahora respecto a lo de hacer variables específicas para los colores de  ciertas cosas como los bordes.. me parece bien; en otras no como las del search bar por ejemplo. Pero entiendo tu idea y me gusta. Esto generalmente se traduce en un tema, el cual normalmente se genera con algún otro tool o librería. Podemos dejar estas de momento pero sí incluyamos los colores como te dije arriba. :D